server/README.md

```diff
@@ -42,14 +96,6 @@ The first param to the container is the name of the desired configuration. For e
 
     docker run -it jboss/infinispan-server cloud -Djboss.default.jgroups.stack=google -Djgroups.google.bucket=... -Djgroups.google.access_key=... 
 
-## Configuring authentication
-   
-The 'default' and 'standalone' running modes don't not have credentials set. In order to define them, run after launching the container:
-
-    docker exec -it $(docker ps -l -q) /opt/jboss/infinispan-server/bin/add-user.sh
-
-and follow the instructions.
-
 ## Running domain mode
 
 Domain mode is composed of a lightweight managing process that does not hold data called domain controller plus one or more
```

Comment:
Running domain mode without setting a password now gives an exception:

```
[Host Controller] Caused by: javax.xml.stream.XMLStreamException: ParseError at [row,col]:[12,21]
[Host Controller] Message: "WFLYCTL0113: '' is an invalid value for parameter value. Values must have a minimum length of 1 characters"
[Host Controller]       at org.jboss.as.controller.SimpleAttributeDefinition.parse(SimpleAttributeDefinition.java:161)
[Host Controller]       at org.jboss.as.controller.SimpleAttributeDefinition.parseAndSetParameter(SimpleAttributeDefinition.java:186)
[Host Controller]       at org.jboss.as.domain.management.parsing.ManagementXml_4.parseSecret(ManagementXml_4.java:528)
[Host Controller]       at org.jboss.as.domain.management.parsing.ManagementXml_4.parseServerIdentities(ManagementXml_4.java:503)
[Host Controller]       at org.jboss.as.domain.management.parsing.ManagementXml_4.parseSecurityRealm(ManagementXml_4.java:453)
[Host Controller]       at org.jboss.as.domain.management.parsing.ManagementXml_4.parseSecurityRealms(ManagementXml_4.java:421)
[Host Controller]       at org.jboss.as.domain.management.parsing.ManagementXml_4.parseManagement(ManagementXml_4.java:176)
```

We should:

1) Make mandatory setting ```-e "MGMT_USER=...." -e "MGMT_PASS=..."``` when using domain mode (and don't use auto-generation, nor print the banner)
2) Mention in this section that both the domain-controller AND the host-controller must be started with the MGMT_USER and MGMT_PASS variables